deployment/ansible/roles/install_docker/tasks/RedHat.yml

```diff
@@ -43,7 +43,7 @@
 
 - name: Install Docker Compose
   ansible.builtin.get_url:
-    url: https://github.com/docker/compose/releases/download/v2.13.0/docker-compose-Linux-x86_64
+    url: https://github.com/docker/compose/releases/download/v{{ install_docker_compose_version }}/docker-compose-Linux-x86_64
```

Comment:
put this url into vars along side with  **install_docker_compose_version**, and make **var** from two vars **url**+**version**

Reply:
Done. 
cool refactoing